coordinator/manifest/manifest.go

```diff
@@ -62,7 +62,11 @@ func (m Manifest) Check(ctx context.Context, zaplogger *zap.Logger) error {
 	// if len(m.Infrastructures) <= 0 {
 	// 	return errors.New("no allowed infrastructures defined")
 	// }
-	for _, marble := range m.Marbles {
+	for idx, marble := range m.Marbles {
+		if marble.Parameters == nil {
```

Comment:
Unit test is missing. In general, write a unit test and make sure it fails before fixing a bug. (See our workflow.)